lib/rules/selector-no-vendor-prefix/index.js

```diff
@@ -41,6 +42,8 @@ const rule = (primary, secondaryOptions, context) => {
 		}
 
 		root.walkRules((ruleNode) => {
+			if (!hasPrefix(ruleNode?.selector)) return;
```

Comment:
[suggest] Is `ruleNode` always truthy, right?

```suggestion
			if (!hasPrefix(ruleNode.selector)) return;
```